test/python/test_extensions_standard.py

```diff
@@ -52,182 +52,424 @@
 from .common import QiskitTestCase
 
 
-class TestStandard(QiskitTestCase):
-    """Standard Extension Test."""
+class StandardExtensionTest(QiskitTestCase):
```

Comment:
Would it be possible to add human-readable docstrings to the assertions methods in this class, ideally describing the parameters as well?

Reply:
fixed in ff05123